backend/routes/query_logs.py

```
@@ -149,23 +149,25 @@ async def clear_query_logs(request: Request, session: dict = Depends(require_adm
 
 
 @router.get("/query-logs/download")
-async def download_query_logs(session: dict = Depends(require_admin_auth)) -> FileResponse:
+async def download_query_logs(session: dict = Depends(require_admin_auth)) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused function argument: `session`

(ARG001)